src/LowercaseSentenceTokenizer.java

```diff
@@ -29,8 +30,31 @@ public class LowercaseSentenceTokenizer implements Tokenizer {
    * @return a list of tokens, where each token is a word or a period
    */
   public List<String> tokenize(Scanner scanner) {
-    // TODO: Implement this function to convert the scanner's input to a list of words and periods
-    return null;
+    // Completed: TODO: Implement this function to convert the scanner's input to a list of words and periods
+
+    // Instantiates a new String ArrayList to hold all the tokens/words from Scanner
+    List<String> words = new ArrayList<>();
+
+    // Loops through the Scanners input as long as there are more lines
+    while (scanner.hasNextLine()) {
+      // Reads in and saves each work/token in the word variable using scanner and converting it to lowercase
+      String word = scanner.next().toLowerCase();
+      
+      // Checks whether a word/token ends with a period using the endsWith method
+      if (word.endsWith(".")) {
```

Comment:
I like that you're adding comments, but you can consider leaving out ones that are a bit redundant here and throughout your program. It’s already clear what the line of code does, this comment doesn’t help our understanding.